mirascope/core/anthropic/call_response_chunk.py

```diff
@@ -66,6 +68,13 @@ def finish_reasons(self) -> list[str] | None:
             return [str(self.chunk.message.stop_reason)]
         return None
 
+    @property
+    def tool_call(self) -> ToolUseBlock | None:
```

Comment:
same thing here, we shouldn't add this, particularly because it doesn't match (e.g. OpenAI is `tool_calls` but this is `tool_call`.

If we find we need this convenience utility enough we should just add a utility function rather than on the call response chunk class.

Reply:
done